nemo/collections/asr/metrics/rnnt_wer_bpe.py

```diff
@@ -202,6 +203,9 @@ def __init__(self, decoding_cfg, decoder, joint, tokenizer: TokenizerSpec):
             decoding_cfg=decoding_cfg, decoder=decoder, joint=joint, blank_id=blank_id + joint.num_extra_outputs
         )
 
+        if isinstance(self.decoding, rnnt_beam_decoding.BeamRNNTInfer):
+            self.decoding.set_decoding_type('subword')
+
```

Comment:
What about adding the Tokenizer here ? And also the vocabulary for char models ?